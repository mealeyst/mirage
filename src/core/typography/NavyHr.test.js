import React from 'react'
import 'jest-styled-components'

import { NavyHr } from 'SRC'

const { shallowWithTheme } = global

describe('(Styled Component) NavyHr', () => {
  const createNavyHr = (props) => {
    return shallowWithTheme(<NavyHr {...props} />)
  }

  test('matching the snapshot', () => {
    expect(createNavyHr())
    .toMatchSnapshot()
  })
})
