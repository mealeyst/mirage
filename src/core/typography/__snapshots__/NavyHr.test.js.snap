// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Styled Component) NavyHr matching the snapshot 1`] = `
.c0 {
  background-color: #00003C;
}

<styled.hr
  className="c0"
  width="100%"
/>
`;
